controller/api/api.go

```diff
@@ -612,8 +612,14 @@ func (a *Api) scaleContainer(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := a.manager.ScaleContainer(containerId, numInstances); err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+	result := a.manager.ScaleContainer(containerId, numInstances)
+	if len(result.Errors) > 0 {
+		http.Error(w, string(json), http.StatusInternalServerError)
+		return
+	}
+
+	if err := json.NewEncoder(w).Encode(result); err != nil {
+		http.Error(w, err.String(), http.StatusInternalServerError)
```

Comment:
Ah man, screwed up here -- rushing too much to get my other PR rebased, so sorry about this.  Will get this fixed and tested tonight!